.travis.yml

```diff
@@ -21,23 +26,24 @@ cache:
 install:
   - if [[ $DEV_LIB_SKIP =~ composer ]]; then composer install --no-dev; fi
```

Comment:
@westonruter We now require `composer install` on all builds.

Reply:
Good catch. I should have removed this.